experimental/grpc_attestation/src/main.rs

```diff
@@ -82,10 +81,10 @@ pub struct Opt {
 // Example message sent to the client.
 const EXAMPLE_MESSAGE: &str = "Example message";
 
-fn request_handler(request: &[u8]) -> Vec<u8> {
-    let parsed_request = String::from_utf8(request.to_vec()).expect("Couldn't parse request");
+async fn request_handler(request: Vec<u8>) -> anyhow::Result<Vec<u8>> {
+    let parsed_request = String::from_utf8(request.to_vec()).context("Couldn't parse request")?;
```

Comment:
Is copying the request into a new vector required here?

Reply:
No. Fixed.